server/channels/store/storetest/draft_store.go

```diff
@@ -275,24 +275,6 @@ func testGetDraft(t *testing.T, ss store.Store) {
 		assert.Equal(t, draft2.Message, draftResp.Message)
 		assert.Equal(t, draft2.ChannelId, draftResp.ChannelId)
 	})
-
-	t.Run("get draft including deleted", func(t *testing.T) {
```

Comment:
Ended up deleting this test since we are hard deleting drafts from the DB. Is there any further cleanup we should do here? We can remove the `includeDeleted` field, and maybe even the `DeleteAt` column altogether?